### PR TITLE
Added uptime diagnostic to status engine

### PR DESCRIPTION
**Pull request by reinkrul**
Nice to have

### CODE REVIEW

File: core/status.go [hunk 1]
```diff
@@ -23,20 +23,23 @@ import (
...
+	system    *System
+	startTime time.Time
 }
 
 //NewStatusEngine creates a new Engine for viewing all engines
 func NewStatusEngine(system *System) Engine {
 	return &status{
-		system: system,
+		system:    system,
+		startTime: time.Now(),
```
Comment (woutslakhorst) [inline core/status.go:42]: I just read something about Monotonic clocks. It seems that the golang Time does both correct! 😱

**Review by woutslakhorst — APPROVED** [review]
Could also be a metric, but that would be quite boring

File: core/status.go (hunk 1 above)
Comment (reinkrul) [reply core/status.go:42]: Nice, didnt realize that's a thing. Question is however, what 'correct' is; if your goal is to display a time relative to the absolute "world time" (e.g. signing time for a document), I'd use the wall clock. For this use case (server up for XXX seconds) I'd use the monotonic clock.

**Comment by reinkrul** [comment]
> Could also be a metric, but that would be quite boring

Just found out it sort of already is;

```
# HELP process_start_time_seconds Start time of the process since unix epoch in seconds.
# TYPE process_start_time_seconds gauge
process_start_time_seconds 1.61407818789e+
```

Not as human-readable though.

**Pull request merged**